File: lib/model/describe_customer.dart
class DescribeCustomer {
     String? email;
     String? phone;
     Map<String, dynamic>? custom;
          
  DescribeCustomer({this.email,this.phone, this.custom});

  Map<String, dynamic> toJson() => {
        'email': email,
        'phone': phone,
        'custom': custom,
      };
}

File: android/src/main/kotlin/com/example/kustomer_native_plugin/models/DescribeCustomer.kt
package com.example.kustomer_native_plugin.models

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import kotlinx.parcelize.RawValue
import java.io.Serializable

@Parcelize
class DescribeCustomer(@SerializedName("email")
                        @Expose
                        val email: String,
                       @SerializedName("phone")
                       @Expose
                       val phone: String,
                       @SerializedName("custom")
                        @Expose
                        val custom: @RawValue Map<String,Any>?,) : Serializable, Parcelable
